cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -50,30 +50,49 @@
   </div>
 
   <!-- LOGS TIMELINE -->
-  <my-timeline timeline-data="LogViewer.data"></my-timeline>
+  <my-timeline
+    namespace-id="{{LogViewer.namespaceId}}"
+    app-id="{{LogViewer.appId}}"
+    program-type="{{LogViewer.programType}}"
+    program-id="{{LogViewer.programId}}"
+    run-id="{{LogViewer.runId}}">
+  </my-timeline>
+
+  <!-- NO DATA MESSAGE -->
+  <div ng-if="LogViewer.data.length === 0" class="jumbotron">
+    Oops! Looks like there are no logs to be found!
+  </div>
 
   <!-- LOGS TABLE -->
-  <div class="logs-table">
-    <table class="table table-bordered">
+  <div ng-if="LogViewer.data.length > 0" class="logs-table">
+    <table class="table table-bordered scroll scroll-table">
       <thead>
-        <th class="time log-header"
+        <th class="time time-header log-header"
             ng-if="LogViewer.configOptions.time && !LogViewer.hiddenColumns.time">
           Time
         </th>
 
-        <th class="level log-header header-level"
+<!--         <th class="level level-header log-header"
             ng-if="LogViewer.configOptions.level && !LogViewer.hiddenColumns.level">
-          <span uib-dropdown>
-            <span uib-dropdown-toggle>Level <span class="fa fa-caret-down"></span></span>
-            <ul uib-dropdown-menu ng-click="$event.stopPropagation();">
-              <li ng-repeat="option in LogViewer.logEvents">
-                <input type="checkbox" ng-click="LogViewer.includeEvent(option)"><span>{{option}}</span></li>
-              </li>
-            </ul>
+          <span class="level-container">
+            <div class='level-text'>Level</div>
+            <div uib-dropdown class='level-dropdown'>
+              <span uib-dropdown-toggle><span class="fa fa-caret-down"></span></span>
+              <ul uib-dropdown-menu ng-click="$event.stopPropagation();">
+                <li ng-repeat="option in LogViewer.logEvents">
+                  <input type="checkbox" ng-click="LogViewer.includeEvent(option)"><span>{{option}}</span></li>
+                </li>
+              </ul>
+            </div>
           </span>
         </th>
+ -->
+        <th class='level2 level2-header'
```

Comment:
since you are commenting out the first one, change this class name to be just level instead of level2